packages/http-client-csharp/generator/TestProjects/CadlRanch/http/type/model/visibility/_Type.Model.Visibility.sln

```diff
@@ -0,0 +1,48 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why is this class called _Type.Model?

Reply:
This is because Type is a reserved keyword so the generator prefixes with an underscore.

Reply:
oh makes sense - thank you!